model/project.go

```diff
@@ -321,8 +330,11 @@ type BuildVariant struct {
 	CronBatchTime string `yaml:"cron,omitempty" bson:"cron,omitempty"`
 
 	// If Activate is set to false, then we don't initially activate the build variant.
-	Activate  *bool `yaml:"activate,omitempty" bson:"activate,omitempty"`
-	PatchOnly *bool `yaml:"patch_only,omitempty" bson:"patch_only,omitempty"`
+	Activate       *bool `yaml:"activate,omitempty" bson:"activate,omitempty"`
+	Patchable      *bool `yaml:"patchable,omitempty" bson:"patchable,omitempty"`
+	PatchOnly      *bool `yaml:"patch_only,omitempty" bson:"patch_only,omitempty"`
```

Comment:
I think we should add the rest of the tags to the docs here since we list them all out
https://github.com/Kimchelly/evergreen/blob/main/docs/02-Configure-a-Project/01-Project-Configuration-Files.md?plain=1#L290